src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookConsumer.java

```diff
@@ -98,7 +73,11 @@ private void triggerJob(RefsChangedWebhookEvent event,
 
     private static Collection<? extends SCM> getScms(ParameterizedJobMixIn.ParameterizedJob<?, ?> job) {
         SCMTriggerItem triggerItem = SCMTriggerItem.SCMTriggerItems.asSCMTriggerItem(job);
-        if (triggerItem != null) {
+        if (triggerItem instanceof WorkflowJob) {
+            if (((WorkflowJob) triggerItem).getDefinition() instanceof CpsScmFlowDefinition) {
```

Comment:
/nit - Could you split the typecasting here and below? It is difficult to do a cursory read and know which class has which method.